flepimop/gempyor_pkg/src/gempyor/config_validator.py

```diff
@@ -52,7 +52,7 @@ def plugin_filecheck(cls, values):
         method = values.get('method')
         plugin_file_path = values.get('plugin_file_path')   
         if method == 'plugin' and not plugin_file_path:
-            raise ValueError('Error in InitialConditions: a plugin file path is required when method is plugin')
+            raise ValueError(f'Error in InitialConditions: a plugin file path is required when method is plugin.')
```

Comment:
Previous file uses double quotes, this one is using singles. Let's converge on a consistent choice.

Reply:
Also don't appear to have added anything parsed here. I guess we're just standardizing on always using `f"..."`?

Reply:
I agree about consistency on both counts (definitely was an accident with the single quotes). But, yeah, that's why I went ahead and still used an f-string.